### PR TITLE
Makes form code more readable

### PR DESCRIPTION
**Pull request by juho-jaakkola**
- Moves starting and closing tags on the same row, so it's easier to see how the elements are nested
- Reduces mixed HTML/PHP
- Uses more descriptive variable names
- Adds margin to bottom, so the captcha won't be so close to submit button

(The changes are clearer using Github's split view instead of unified view)

### CODE REVIEW

**Comment by juho-jaakkola** [comment]
I fixed the typo that caused Scrutinizer to fail the tests.

File: views/default/input/captcha.php [hunk 1]
```diff
@@ -28,50 +28,58 @@
...
-	    float: left;
-	}   
-	
+	#image_captcha {
+		margin: 15px 0;
+	}
+
 	#image_captcha .img {
-		cursor:pointer;
-		display:none;
```
Comment (jdalsem) [inline views/default/input/captcha.php]: is `display: none` no longer needed? wasn't it there for no-script support?

Comment (juho-jaakkola) [inline views/default/input/captcha.php]: How can user see the image if there's no script that makes it visible?

Comment (jdalsem) [inline views/default/input/captcha.php]: they will see the radio button (that is hidden if javascript is enabled)

**Pull request merged**